doc/user/content/sql/system-catalog.md

```diff
@@ -56,19 +56,42 @@ schema.
 The `mz_array_types` table contains a row for each array type in the system.
 
 Field          | Type       | Meaning
----------------|------------|----------
+---------------|------------|--------
 `type_id`      | [`text`]   | The ID of the array type.
 `element_id`   | [`text`]   | The ID of the array's element type.
 
+### `mz_arrangement_sharing`
+
+The `mz_arrangement_sharing` source describes how many [dataflows][dataflow]
+share each [arrangement] in the system.
```

Comment:
It's actually how many uses of each arrangement are there, so if there is one dataflow that uses an arrangement five times, it will contribute five. I'm happy to edit nits like this (and will switch modes if there are a bunch more).

Reply:
No this is perfect, thanks! Fixed.